packages/terra-show-hide/src/ShowHide.jsx

```diff
@@ -56,17 +62,33 @@ const ShowHide = (props) => {
     preview,
     intl,
     isOpen,
+    focusRef,
     ...customProps
   } = props;
 
   const theme = React.useContext(ThemeContext);
-  const ref = React.useRef(null);
+  const contentRef = React.useRef(null);
 
+  /**
+   * Upon showing hidden content, add required attributes and set focus to the child element provided by focusRef,
+   * or the content container element, if focusRef wasn't provided.
+   */
   React.useEffect(() => {
-    if (isOpen && ref?.current) {
-      ref.current.focus();
+    if (isOpen) {
+      let element = null;
+      if (props.focusRef?.current) {
+        element = props.focusRef?.current;
+      } else if (contentRef?.current) {
+        element = contentRef?.current;
+      }
+      if (element) {
+        element.setAttribute('tabIndex', '-1');
+        element.setAttribute('data-focus-styles-enabled', isOpen);
+        element.setAttribute('role', 'group');
+        element.focus();
+      }
     }
-  }, [isOpen]);
+  }, [isOpen, props.focusRef, contentRef]);
```

Comment:
I think we can just use `focusRef` since we destructured the props above.

Reply:
Good catch, removed: https://github.com/cerner/terra-core/pull/3838/commits/ccf7a43bde31fc3eedf2b274cc4a2d5f9249f738